docs/source/resources/scyllaclusters/basics.md

```diff
@@ -0,0 +1,123 @@
+  name: scylladb-config
+data:
+  scylla.yaml: |
+    authenticator: PasswordAuthenticator
+    authorizer: CassandraAuthorizer
+    # Other options
+EOF
+:::
+
+:::{note}
+Some of the ScyllaDB config is also generated by the {{productName}} based on your ScyllaCluster definitions.
+While you shall not define conflicting options here ({{productName}} config wins), we still want to give you a reasonable control to fine tune some ScyllaDB knobs.
+IOW. please stay away from touching networking, listen or published addresses and so on, but feel free to tune buffer sizes and such.
+:::
+
+Now we can create a simple ScyllaCluster to get ScyllaDB running.
+
+:::{code-block} bash
+:linenos:
+:substitutions:
+
+kubectl apply --server-side -f=<<EOF
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: scylladb
+spec:
+  repository: {{imageRepository}}
+  version: {{imageTag}}
+  agentVersion: {{agentVersion}}
+  developerMode: false
+  automaticOrphanedNodeCleanup: true
+  sysctls:
+  - fs.aio-max-nr=30000000
+  datacenter:
+    name: us-east-1
+    racks:
+    - name: us-east-1a
+      members: 1
+      scyllaConfig: scylladb-config
+      storage:
+        capacity: 100Gi
+        storageClassName: scylladb-local-xfs
+      resources:
+        requests:
+          cpu: 1
+          memory: 8Gi
+        limits:
+          cpu: 1
+          memory: 8Gi
+EOF
+:::
+
+:::{note}
+This values in this examples are only illustratory.
+You should always adjust the resources and storage capacity depending on your needs or the size and the type of your Kubernetes nodes.
+:::
+
+:::{include} ../../.internal/tuning-qos-caution.md
+:::
+
+(scyllacluster-enterprise)=
+::::{note}
+{{productName}} works with both ScyllaDB Open Source and ScyllaDB Enterprise.
+You only have to adjust the [repository and tag fields](api-scylla.scylladb.com-scyllaclusters-v1-.spec) for each ScyllaCluster.
+
+In addition to it, if you want to use tuning from the Enterprise repository, you have to adjust [**scyllaUtilsImage** on the global ScyllaOperatorConfig/cluster](../scyllaoperatorconfigs.md#tuning-with-scylladb-enterprise).
+
+:::{code-block} bash
+:linenos:
+:substitutions:
+:emphasize-lines: 6,7
+
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: scylladb
+spec:
+  repository: {{enterpriseImageRepository}}
+  version: {{enterpriseImageTag}}
+  # ...
+EOF
+:::
+
+::::
+
+## Forcing a rolling restart
+
+When you make a change to ScyllaDB config to an option that's not live reloaded by ScyllaDB, or want to trigger a rolling restart for a different reason, ScyllaCluster allow triggering the rolling restarts declaratively by changing `ScyllaCluster.spec.forceRedeploymentReason` to any other value. This will trigger a rolling restart of all ScyllaDB nodes in sequence, always respecting the [PodDistruptionsBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) and keeping the cluster available.
```

Comment:
```suggestion
When you change a ScyllaDB config option that's not live reloaded by ScyllaDB, or want to trigger a rolling restart for a different reason, ScyllaCluster allows triggering the rolling restarts declaratively by changing `ScyllaCluster.spec.forceRedeploymentReason` to any other value. This will trigger a rolling restart of all ScyllaDB nodes in sequence, always respecting the [PodDistruptionsBudget](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#pod-disruption-budgets) and keeping the cluster available.
```

Reply:
done